paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundSettingDefinition.kt

```diff
@@ -4,34 +4,65 @@ import com.stripe.android.paymentsheet.PaymentSheet
 import com.stripe.android.paymentsheet.example.playground.PlaygroundState
 import com.stripe.android.paymentsheet.example.playground.model.CheckoutRequest
 
-internal interface PlaygroundSettingDefinition<T> {
-    val defaultValue: T
+internal sealed interface PlaygroundSettingDefinition<T> {
```

Comment:
I don't think the changes in this file are necessary. 

This type is already quite complex, and we're shoving more responsibility onto it, and abstracting things I'm not sure are necessary.

I assumed we'd we'd make `IntegrationTypeSettingDefinition` not part of _any_ of this, and just manually set it with non generics.